server/src/Types.hs

```diff
@@ -0,0 +1,2 @@
+module Types where
+
```

Comment:
This module is empty and can be deleted.

Very nice. I don't like god modules.